flask_project/campaign_manager/views.py

```diff
@@ -56,37 +65,99 @@
 
 MAX_AREA_SIZE = 320000000
 
+# create session for Database Queries
+Session = sessionmaker(bind=engine)
+Session.configure(bind=engine)
+session = Session()
+session.rollback()
+
+
+@campaign_manager.route('/add_osm_user/<username>', methods=['GET', 'POST'])
+def add_osm_user(username):
+    user = session.query(User).filter(User.osm_user_id == username).all()
+    # add to db if user is new.
+    if len(user) == 0:
+        new_user = User(osm_user_id=username, email='')
+        session.add(new_user)
+        session.commit()
+    return "user is here"
```

Comment:
We should return a JSON object or something else than just a string. Even a response with a status code 200 should be enough.